docs/Introduction.iOS.md

```diff
@@ -41,10 +42,14 @@ Whether you’ve selected to apply the configuration in a  `.detoxrc.json` or bu
 }
 ```
 
+:::info
+
 For a comprehensive explanation of Detox configuration, see our [dedicated API-reference guide](APIRef.Configuration.md).
 
+:::
+
 In the above configuration example, make sure to provide the correct information for your project/app. Under the key `"binaryPath"`, you should provide the path of the .app bundle to use. Normally, this is the path where the `"build”` command would output this bundle. Under the key `"build"`, specify the `xcodebuild` command for your project.
```

Comment:
@asafkorem , I wonder if you can take a screenshot of Xcode starting the build of the project and imply that a user can copy the command from there? Not critical, but still I couldn't help but ask.

Reply:
I can't, the `xcodebuild` command is not invoked by Xcode when building the app from there.